content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,28 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.12 [2020-06-12]
+
+### Features
+
+- Make all `pkg` applications stateful through stacks.
+- Add ability to export a stack's existing resource state as a `pkg`.
+- Update influx `pkg` commands with improved usage and examples in long form.
+- Extend `influx` CLI with version and User-Agent.
+- Integrate `RedirectTo` functionality so Cloud users are redirected back to the originally linked page after login.
```

Comment:
What is the originally linked page..the last page a user was on before they logged off the last time?

Reply:
@Asalem1 can you help provide some insight here

Reply:
The originally linked page would refer to the page that the user was trying to originally access before getting redirected to the login page. Previously, it doesn't matter where the user was trying to access before they logged in, the context to that original link was lost and the user was directed to the home page. Now, if a user clicks on the link:

https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/data-explorer

and gets redirected to the login page, they will be redirected to the link they originally tried to access after logging in